File: tests/fixtures/fixture_java_unsafe/Bad.java
import java.io.IOException;

public class Bad {
    // TODO: replace Runtime.exec with ProcessBuilder
    public static void main(String[] args) {
        System.out.println("starting");
        try {
            Runtime.getRuntime().exec("ls -la");
        } catch (IOException e) {}
        try {
            Class.forName("com.foo.Bar");
        } catch (Exception e) {
            e.printStackTrace();
        }
        try {
            Thread.sleep(100);
        } catch (InterruptedException e) {}
    }
}

File: tests/fixtures/fixture_kotlin_unsafe/Bad.kt
// TODO: drop the not-null assertions
package demo

fun bad(name: String?) {
    println("hello $name")
    val length = name!!.length
    Runtime.getRuntime().exec("ls -la")
    try {
        // op
    } catch (e: Exception) {
    }
    TODO("not implemented")
}
